code/ModelAdmin.php

```diff
@@ -274,6 +274,13 @@ public function getEditForm($id = null, $fields = null)
         $form->setFormAction($editFormAction);
         $form->setAttribute('data-pjax-fragment', 'CurrentForm');
 
+        // Sudo mode
```

Comment:
Scenario is a managed model on a model admin, for instance Member on SecurityAdmin (which extends ModelAdmin)